frontend/src/components/templates/SearchModal.tsx

```diff
@@ -13,6 +15,21 @@ function SearchModal({ id, searchId, open, setOpen }: SearchModalProps) {
 	const modalContent = useRef<any>(null);
 	const searchInput = useRef<any>(null);
 
+	const [text, setText] = useState('');
+	const [results, setResults] = useState([]);
+	const headerAuth = {
+		headers: {
+			Authorization: ` Bearer ${localStorage.getItem('access_token')}`,
+		},
+	};
+	const searchData = async () => {
+		const resp = await axios.get(`http://localhost:3001/search`, { params: { search: `${text}` }, ...headerAuth });
```

Comment:
@Mittensdlara You should provide the URL using env. not hard coded.

Reply:
I changed the `url`